<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <title>Vaitea Opuu</title>
        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
        <link href="product.css" rel="stylesheet">
    </head>
    <body>
        <nav class="site-header sticky-top py-1 bg-dark">
            <div class="container d-flex flex-column flex-md-row justify-content-between" style="background-image: url('img/glms.png'); background-position: left; background-size: 30%">
                <a class="py-2" href="#">
                    <strong>Vaitea Opuu, Ph.D</strong><br/>
                    vaiteaopuu@gmail.com
                </a>
                <a class="py-2 d-none d-md-inline-block" href="#research_int">Research interests</a>
                <a class="py-2 d-none d-md-inline-block" href="#short_bio">Short Bio</a>
                <a class="py-2 d-none d-md-inline-block" href="#publications">Papers</a>
            </div>
        </nav>
        <div class="container-fluid bcontent">
            <div class="row">
                <div class="container" style="background-image: img/md_res.png">
                    <div class="social-icons primary-text clearfix" style="display: flex;">
                        <table class="table">
                            <tr>
                                <td> <a href="https://www.researchgate.net/profile/Vaitea-Opuu"><img src="img/researchgate-logo-svgrepo-com.svg" style="max-width: 40px" alt=""/></a> </td>
                                <td> <a href="https://github.com/vaiteaopuu"><img src="img/github-icon-1-logo-svgrepo-com.svg" style="max-width: 50px" alt=""/></a> </td>
                                <td> <a href="cv/cv.pdf"><img src="img/cv-file-interface-symbol-svgrepo-com.svg" alt=""/></a> </td>
                                <td> <a href="https://scholar.google.com/citations?user=QjPCEicAAAAJ&hl=en&oi=ao"><img src="img/google-scholar-svgrepo-com.svg" style="max-width: 40px" alt=""/></a> </td>
                            </tr>
                        </table>
                    </div>
                    <hr class="mt-2 mb-3"//>
                    <div class="bg-dark d-md-flex flex-md-equal w-100 my-md-3
                                pl-md-3" style="background-image:
                                url('img/land_4.png'); background-position:
                                right; background-size: 100%; background-repeat:
                                no-repeat;">
                        <div class="mr-md-3 pt-3 px-3 pt-md-5 px-md-5 text-left text-white overflow-hidden">
                            <div class="my-3 py-3">
                                <h2>Greetings!</h2>
                                <p>
                                    I'm a computational biologist working as a
                                    postdoc in <a href=https://www.lbe.espci.fr/home/>ESPCI</a>.
                                </p>

                                <p>
                                    How does a protein or an RNA works? How
                                    could I make one that does this interesting
                                    thing? ...mmmhhh... That is the type of
                                    question I'm working on.
                                </p>
                            </div>
                        </div>
                        <div class="mr-md-3 pt-3 px-3 pt-md-5 px-md-5 text-center overflow-hidden">
                            <div class="my-3 p-3">
                                <img src="img/vopuu.png" alt="Avatar" style="max-width: 40%;border-radius: 100%"/>
                            </div>
                        </div>
                    </div>

                    <section id="research_int">
                        <h2><strong style="font-size:30px">&#127891;</strong> Research interests</h2>
                        <div class="position-relative overflow-hidden p-3 p-md-5 m-md-3" style="background-image: url('img/free_nrj_comb.png'); background-position: right; background-size: 30%; background-repeat: no-repeat;">
                            <h4> How to design proteins ? </h4>
                            <p style="max-width: 70%">
                                Don't let the simplicity of this question fool
                                you! Indeed, protein design hides multiple
                                fundamental questions - which essentially shows
                                how little we know about these very well-known
                                molecules. well-known molecules.
                            </p>
                            <p style="max-width: 70%">
                                Roughly speaking, two approaches are fighting:
                                i) machine learning takes natural protein
                                examples to design new ones, and ii) physics
                                explicitly models the details hidden in protein
                                atoms! I followed the second option, where in
                                collaboration with experimentalists, we not only
                                designed proteins for molecular structure
                                stability but also new catalytic proteins
                                (enzymes). The methods we worked on are
                                implemented in the <a
                                href="https://proteus.polytechnique.fr/">Proteus</a>
                                Software developed in <a
                                href="https://bioc.polytechnique.fr/biocomputing/">Thomas
                                Simonson's lab</a>.
                            </p>
                        </div>
                        <div class="position-relative overflow-hidden p-3 p-md-5 m-md-3" style="background-image: url('./img/overlapping_reading_frames.png'); background-position: right; background-size: 30%; background-repeat: no-repeat;">
                            <h4> One gene can hide another one ! </h4>
                            <p style="max-width: 70%">
                                There is a natural process by which one gene can
                                hide another gene &#8212; these genes are called
                                overlapping genes. This sounds like magic, but
                                it's not! This is simply due to two features of
                                the genetic code: DNA is read by codons
                                (triplets of nucleotides), and the genetic code
                                is redundant.
                            </p>
                            <p style="max-width: 70%">
                                Overlapping genes have multiple implications in
                                evolutionary theories but not only. We devised an
                                algorithm that can somehow "morph" two genes such that
                                in the end both are intricated (overlap). You could only
                                diferenciate the first gene from the second by reading
                                the genetic sequence in a special way. Bluffing, right ?
                            </p>
                        </div>
                    </section>
                    <br/><br/>

                    <section id="short_bio">
                        <h2><strong style="font-size:30px">&#127891;</strong> Short Bio</h2>
                        <div class="position-relative overflow-hidden p-3 p-md-5 m-md-3">
                            <p>
                                After a Master in Bioinformatics at Paris-Saclay
                                University (2015-2017), I started a PhD in Biology at <a href="https://www.polytechnique.edu/">Ecole Polytechnique</a> (<a href="https://www.ip-paris.fr/en">Institut
                                Polytechnique de Paris</a>). From 2017 to 2020, I
                                worked on Computational Protein Design under the
                                supervision of Pr. Thomas Simonson in the <a href="https://portail.polytechnique.edu/bioc/en/presentation/presentation">
                                Structural Biology of the Cell</a> laboratory. Next, I
                                started a postdoc position at the <a href="https://www.mis.mpg.de">Max Planck
                                Institute for Mathematics in Sciences</a> (Leipzig, Germany),
                                from 2020 to 2022.
                            </p>
                        </div>
                    </section>

                    <section id="publications">
                        <h2><strong href="font-size:30px">&#128218;</strong> Publications</h2>
                        <table class="table">
                            <tbody>
                                <tr>
                                    <td>
                                    2022
                                    </td>
                                </tr>
                                <tr>
                                    <td>
                                        <span class="badge badge-secondary">Conference</span>
                                        <b>Investigating graph neural network for RNA structural embedding </b><br>
                                        <i><b>Machine Learning for Structural Biology Workshop, NeurIPS 2022</b>, 2022,</i> <a href="https://www.mlsb.io/papers_2022/Investigating_graph_neural_network_for_RNA_structural_embedding.pdf">paper</a> <br>
                                        Vaitea Opuu, Hélène Bret <br>
                                    </td>
                                </tr>

                                <tr>
                                    <td>
                                        <span class="badge badge-primary">Journal</span>
                                        <b>RAFFT: Efficient prediction of RNA folding pathways using the fast Fourier transform</b><br>
                                        <i><b>Plos Computational Biology</b>, 2022,</i> <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010448">doi</a> <br>
                                        Vaitea Opuu, Nono S. C. Merleau, Vincent Messow, Matteo Smerlak <br>
                                    </td>
                                </tr>

                                <tr>
                                    <td>
                                        <span class="badge badge-info">Chapter</span>
                                        <b>Knowledge-Based Unfolded State Model for Protein Design</b><br/>
                                        <i><b>methods in Molecular Biology: Computational Peptide Science</b>, 2022</i><br/>
                                        Vaitea  Opuu, David Mignon, Thomas Simonson <br/>
                                    </td>
                                </tr>
                                <tr>
                                    <td>
                                        2020
                                    </td>
                                </tr>


                                <tr>
                                    <td>
                                        <span class="badge badge-primary">Journal</span>
                                        <b>Adaptive Landscape Flattening Allows the Design of Both Enzyme: Substrate Binding and Catalytic Power </b><br>
                                        <i><b>Plos Computational Biology</b>, 2020,</i> <a href="https://doi.org/10.1371/journal.pcbi.1007600">doi</a> <br>
                                        Vaitea Opuu, Giuliano Nigro, Thomas Gaillard, Emmanuelle Schmitt, Yves Mechulam, Thomas Simonson <br>
                                    </td>
                                </tr>

                                <tr>
                                    <td>
                                        <span class="badge badge-primary">Journal</span>
                                        <b>A Physics-Based Energy Function Allows the Computational Redesign of a Pdz Domain </b><br/>
                                        <i><b>Scientific Reports</b>, 2020,</i> <a href="https://doi.org/10.1038/s41598-020-67972-w">doi</a> <br/>
                                        Vaitea Opuu*, Young Joo Sun*, Titus Hou, Nicolas Panel, Ernesto J. Fuentes, Thomas Simonson <br/>
                                    </td>
                                </tr>


                                <tr>
                                    <td>
                                        <span class="badge badge-primary">Journal</span>
                                        <b>Proteus software for physics-based protein design </b><br/>
                                        <i><b>The Journal of Physical Chemistry Part A</b>, 2020,</i> <a href="https://doi.org/10.1021/acs.jpca.0c07605">doi</a> <br/>
                                        David Mignon, Karen Druart*, Eleni Michael*, Vaitea  Opuu*, Savvas Polydorides*, Francesco Villa*, Thomas Gaillard, Nicolas Panel, Georgios Archontis, Thomas Simonson <br/>
                                    </td>
                                </tr>

                                <tr>
                                    <td>
                                        <span class="badge badge-info">Chapter</span>
                                        <b>Computational Design of PDZ-Peptide Binding</b><br/>
                                        <i><b>methods in Molecular Biology: PDZ domains</b>, 2020</i><br/>
                                        Nicolas Panel, Vaitea  Opuu, Francesco Villa, David Mignon, Thomas Simonson <br/>
                                    </td>
                                </tr>
                                <tr>
                                    <td>
                                        2018
                                    </td>
                                </tr>

                                <tr>
                                    <td>
                                        <span class="badge badge-primary">Journal</span>
                                        <b>Meet-U: educating through research immersion </b><br/>
                                        <i><b>Plos Computational Biology</b>, 2018</i>, <a href="https://doi.org/10.1371/journal.pcbi.1005992">doi</a> <br/>
                                        Nika Abdollahi, Alexandre Albani, Eric Anthony, Agnes Baud, Mélissa Cardon, Robert Clerc, Dariusz Czernecki, Romain Conte, Laurent David, Agathe Delaune, <i>et al</i><br/>
                                    </td>
                                </tr>

                                <tr>
                                    <td>
                                        <span class="badge badge-secondary">Conference</span>
                                        <b>Computational design of fully overlapping coding schemes for portein pairs and triplets</b><br/>
                                        <i><b>(proceeding of) advances in systems and synthetic biology</b>, page 97, 2018</i><br/>
                                        Vaitea Opuu, Martin Silvert, Thomas Simonson <br/>
                                    </td>
                                </tr>

                                <tr>
                                    <td>
                                        2017
                                    </td>
                                </tr>
                                <tr>
                                    <td>
                                        <span class="badge badge-primary">Journal</span>
                                        <b>Computational Design of Fully Overlapping Coding Schemes for Protein Pairs and Triplets </b><br/>
                                        <i><b>Scientific Reports</b>, 2017</i>, <a href="https://doi.org/10.1038/s41598-017-16221-8">doi</a> <br/>
                                        Vaitea Opuu, Martin Silvert, Thomas Simonson <br/>
                                    </td>
                                </tr>


                                <tr>
                                    <td>
                                        <span class="badge badge-warning">Ph.D Thesis</span>
                                        <b>Computational design of proteins and enzymes</b><br/>
                                        <i><b>Institut Polytechnique de Paris</b>, 2020,</i> <a href="https://tel.archives-ouvertes.fr/tel-03082636/">link</a><br/>
                                        Vaitea Opuu<br/>
                                    </td>
                                </tr>
                            </tbody>
                        </table>
                    </section>

                </div>
            </div>
        </div>


        <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
        <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" crossorigin="anonymous"></script>

    </body>
</html>
