<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <title>Vaitea Opuu</title>
        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
    </head>
    <body>

        <!-- <nav class="navbar navbar-dark sticky-top bg-dark flex-md-nowrap p-0" align="left">
             <img src="img/md_res.png" style="max-width: 5%"/>
             </nav> -->

        <div class="container-fluid bcontent">
            <div class="row">
                <nav class="navbar-nav navbar-expand-sm justify-content-center px-3 bg-dark">
                    <div class="sidebar-sticky bg-dark">
                        <ul class="nav nav-pills flex-column bg-dark">
                            <br/>
                            <li class="nav-item"> <a class="nav-link" href="#research_int" style="color: #ffffff"><h5><strong style="font-size:30px">&#127891;</strong> Research interests</h5></a> </li>
                            <br/>
                            <li class="nav-item"> <a class="nav-link" href="#publications" style="color: #ffffff"><h5><strong href="font-size:30px">&#128218;</strong> Publications</h5></a> </li>
                            <!-- <li class="nav-item"> <a class="nav-link" href="#others"><h5>Others</h5></a> </li> -->
                        </ul>
                    </div>
                </nav>

                <div class="container" style="background-image: img/md_res.png">

                    <h1><strong>Vaitea Opuu, Ph.D</strong></h1>
                    <img src="img/vopuu.png" alt="Avatar" style="max-width: 20%;border-radius: 100%"/>

                    <br/><br/>
                    <strong><strong style="font-size:30px"> &#127968; </strong> Inselstraße 22, 04103 Leipzig, Germany</strong><br/>
                    <strong><strong style="font-size:30px"> &#9993; </strong> vopuu@mis.mpg.de </strong><br/> <br/> <br/>

                    <div class="social-icons primary-text clearfix" style="display: flex;">
                        <table class="table">
                            <tr>
                                <td> <a href="https://www.researchgate.net/profile/Vaitea-Opuu"><img src="img/researchgate-logo-svgrepo-com.svg" style="max-width: 40px" alt=""/></a> </td>
                                <td> <a href="https://github.com/vaiteaopuu"><img src="img/github-icon-1-logo-svgrepo-com.svg" style="max-width: 50px" alt=""/></a> </td>
                                <td> <a href="cv/cv.pdf"><img src="img/cv-file-interface-symbol-svgrepo-com.svg" alt=""/></a> </td>
                            </tr>
                        </table>
                    </div>
                    <br/><br/>

                    <p>
                        Hi! I'm a Postdoc in the group <a href="https://www.mis.mpg.de/strevol/index.html">Structure of evolution</a>,
                        at <a href="https://www.mis.mpg.de">the Max Planck
                        Institute for Mathematics in Sciences</a> (Leipzig, Germany),
                        since December 2020. I currently work on computational
                        methods to design RNA molecules. Those methods rely on
                        statistical physics, neural networks, and dynamic
                        programming.
                    </p>

                    <p>
                        After a Master in Bioinformatics at Paris-Saclay University (2015-2017),
                        I started a PhD in Biology at <a href="https://www.polytechnique.edu/">Ecole Polytechnique</a> (<a href="https://www.ip-paris.fr/en">Institut
                        Polytechnique de Paris</a>). From 2017 to 2020, I worked
                        on Computational Protein Design (CPD) under the supervision
                        of Pr. Thomas Simonson in <a href="https://portail.polytechnique.edu/bioc/en/presentation/presentation">the
                        Structural Biology of the Cell</a> laboratory. I defended my PhD on the 29th of
                        October 2020.
                    </p>
                    <br/><br/>

                    <section id="research_int">
                        <h2><strong style="font-size:30px">&#127891;</strong> Research interests</h2>

                        <p>
                            I'm interested in biopolymers such as proteins,
                            DNAs, or RNAs. Most of the works I contributed to
                            were efforts toward the design of biologically
                            relevant proteins. Using molecular mechanics,
                            continuum electrostatics, Monte Carlo simulations,
                            and molecular dynamic simulations, we aimed at
                            understanding the subtle mutational interplays. Such
                            works allow the detailed understanding of the
                            mutational impact on catalysis, even when
                            considering multiple reactions. Those approaches are
                            implemented in the <a href="https://proteus.polytechnique.fr/">Proteus</a> Software developed in
                            <a href="https://bioc.polytechnique.fr/biocomputing/">Thomas Simonson's lab</a>.

                        </p>
                        <br/>
                        <img src="./img/free_nrj_comb.png" alt="Catalysis" style="max-width: 50%;"/>
                        <br/>
                        <br/>

                        <p>
                            In a completely different aspect of bio-polymer
                            design, I worked on the first algorithm for the
                            design of overlapping genes. The algorithm designs
                            one single DNA molecule in which at least two genes
                            are encoded and both genes are overlapping. These
                            genes may encode completely different proteins. This
                            uses a dynamic programming scheme similar to the
                            sequence alignment one.
                        </p>
                        <br/>

                        <img src="./img/overlapping_reading_frames.png" alt="Overlapping frames" style="max-width: 40%;"/>
                        <img src="./img/design_6_clean.png" alt="All overlapping frames" style="max-width: 40%;"/>

                    </section>
                    <br/><br/>

                    <section id="publications">
                        <h2><strong href="font-size:30px">&#128218;</strong> Publications</h2>
                        <table class="table">
                            <tbody>
                                <tr>
                                    <td>
                                        <span class="badge badge-primary">Journal</span>
                                        <b>Adaptive Landscape Flattening Allows the Design of Both Enzyme: Substrate Binding and Catalytic Power </b><br>
                                        <i><b>Plos Computational Biology</b>, 2020,</i> <a href="https://doi.org/10.1371/journal.pcbi.1007600">doi</a> <br>
                                        Vaitea Opuu, Giuliano Nigro, Thomas Gaillard, Emmanuelle Schmitt, Yves Mechulam, Thomas Simonson <br>
                                    </td>
                                </tr>

                                <tr>
                                    <td>
                                        <span class="badge badge-primary">Journal</span>
                                        <b>A Physics-Based Energy Function Allows the Computational Redesign of a Pdz Domain </b><br/>
                                        <i><b>Scientific Reports</b>, 2020,</i> <a href="https://doi.org/10.1038/s41598-020-67972-w">doi</a> <br/>
                                        Vaitea Opuu*, Young Joo Sun*, Titus Hou, Nicolas Panel, Ernesto J. Fuentes, Thomas Simonson <br/>
                                    </td>
                                </tr>


                                <tr>
                                    <td>
                                        <span class="badge badge-primary">Journal</span>
                                        <b>Proteus software for physics-based protein design </b><br/>
                                        <i><b>The Journal of Physical Chemistry Part A</b>, 2020,</i> <a href="https://doi.org/10.1021/acs.jpca.0c07605">doi</a> <br/>
                                        David Mignon, Karen Druart*, Eleni Michael*, Vaitea  Opuu*, Savvas Polydorides*, Francesco Villa*, Thomas Gaillard, Nicolas Panel, Georgios Archontis, Thomas Simonson <br/>
                                    </td>
                                </tr>

                                <tr>
                                    <td>
                                        <span class="badge badge-primary">Journal</span>
                                        <b>Meet-U: educating through research immersion </b><br/>
                                        <i><b>Plos Computational Biology</b>, 2018</i>, <a href="https://doi.org/10.1371/journal.pcbi.1005992">doi</a> <br/>
                                        Nika Abdollahi, Alexandre Albani, Eric Anthony, Agnes Baud, Mélissa Cardon, Robert Clerc, Dariusz Czernecki, Romain Conte, Laurent David, Agathe Delaune, <i>et al</i><br/>
                                    </td>
                                </tr>

                                <tr>
                                    <td>
                                        <span class="badge badge-primary">Journal</span>
                                        <b>Computational Design of Fully Overlapping Coding Schemes for Protein Pairs and Triplets </b><br/>
                                        <i><b>Scientific Reports</b>, 2017</i>, <a href="https://doi.org/10.1038/s41598-017-16221-8">doi</a> <br/>
                                        Vaitea Opuu, Martin Silvert, Thomas Simonson <br/>
                                    </td>
                                </tr>

                                <tr>
                                    <td>
                                        <span class="badge badge-info">Chapter</span>
                                        <b>In Methods in Molecular Biology: PDZ domains </b><br/>
                                        <i><b>methods in Molecular Biology: PDZ domains</b>, 2020</i><br/>
                                        Nicolas Panel, Vaitea  Opuu, Francesco Villa, David Mignon, Thomas Simonson <br/>
                                    </td>
                                </tr>

                                <tr>
                                    <td>
                                        <span class="badge badge-secondary">Conference</span>
                                        <b>Computational design of fully overlapping coding schemes for portein pairs and triplets</b><br/>
                                        <i><b>(proceeding of) advances in systems and synthetic biology</b>, page 97, 2018</i><br/>
                                        Vaitea Opuu, Martin Silvert, Thomas Simonson <br/>
                                    </td>
                                </tr>

                                <tr>
                                    <td>
                                        <span class="badge badge-warning">Ph.D Thesis</span>
                                        <b>Computational design of proteins and enzymes</b><br/>
                                        <i><b>Institut Polytechnique de Paris</b>, 2020,</i> <a href="https://tel.archives-ouvertes.fr/tel-03082636/">link</a><br/>
                                        Vaitea Opuu<br/>
                                    </td>
                                </tr>
                            </tbody>
                        </table>
                    </section>

                </div>
            </div>
        </div>


        <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
        <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" crossorigin="anonymous"></script>

    </body>
</html>
